eq-author/src/components/QuestionnaireTable/TableHead.js

```diff
@@ -107,61 +108,45 @@ const UnsortableTH = styled(TH)`
   padding: 1em;
 `;
 
-const TableHead = (props) => {
-  return (
-    <thead>
-      <tr>
-        <SortableTH
-          sortColumn="title"
-          colWidth="20%"
-          dataTest="title-sort-button"
-          {...props}
-        >
-          Title
-        </SortableTH>
-        <SortableTH
-          sortColumn="createdBy.displayName"
-          colWidth="15%"
-          dataTest="owner-sort-button"
-          {...props}
-        >
-          Owner
-        </SortableTH>
-        <SortableTH
-          sortColumn="createdAt"
-          colWidth="10%"
-          dataTest="created-sort-button"
-          {...props}
-        >
-          Created
-        </SortableTH>
-        <SortableTH
-          sortColumn="updatedAt"
-          colWidth="9%"
-          dataTest="modified-sort-button"
-          {...props}
-        >
-          Modified
-        </SortableTH>
-        <UnsortableTH colWidth="10%">Permissions</UnsortableTH>
+function checkEnabled(enabledRows, headings) {
+  enabledRows.map((heading) => {
+    const headingElement = headings.find(
+      (element) => element.heading === heading
+    );
+    headingElement.enabled = true;
+    return null;
+  });
```

Comment:
May wish to use `forEach` instead for cases like this when you don't actually care about the return value (you're mapping everything => `null` here)

Reply:
Might also find it easier for the general flow of things to `filter` out the non-enabled headings instead of doing this and pass around only the headings you really want to be rendered - would also simplify your downstream code (no need to keep checking if stuff is enabled)